### PR TITLE
Mentioned names restored in export

### LINKED ISSUE #102: Fix export with mentioned names
I initially implemented it before, but it was breaking export so I commented it out. Still need to re-add it in.

### PR DESCRIPTION
**Pull request by ryanctan**
Fixes #102. @epicfaace's previously commented out code looks like it's not breaking anymore

### CODE REVIEW

**Review by epicfaace — CHANGES_REQUESTED** [review]
Try going to Explore and not filtering by anything, then doing an export. Then you get this error 😢 

image

**Comment by ryanctan** [comment]
@epicfaace, new fix pushed, please review when you get the chance

File: query.js [hunk 1]
```diff
@@ -543,6 +543,9 @@ exports.projectForEntryList = projectForEntryList
 
 function parseExport(res) {
 
+    var mentions = [];
```
Comment (epicfaace) [inline query.js:546]: @ryanctan how exactly did these latest changes fix the export?

Comment (ryanctan) [reply query.js:546]: So I actually don't really know why the commented-out code didn't work, I kinda just combined that code with the code for activities and travels and it worked. Sorry I couldn't be more helpful!

Comment (epicfaace) [reply query.js:546]: Hmm, code is way more complicated, but it works, so let's merge!

**Review by epicfaace — APPROVED** [review]

**Pull request merged**